uw-frame-components/portal/widgets/partials/demo-widgets.html

```diff
@@ -5,33 +5,29 @@
 
   <md-content layout-padding>
 
-    <!-- LOCAL DEMO -->
-    <div layout="row" layout-align="start center">
-      <div flex-xs="100">
+    <!--
+      LOCAL DEMO
+      - Must set SERVICE_LOC->widgetApi.entry in app-config.js to: "staticFeeds/"
+    -->
+    <div layout="row" layout-align="center center">
+      <div flex-xs="100" style="max-width:310px">
         <widget fname="sample-widget__search-with-links"></widget>
       </div>
-      <div flex-xs="100">
+      <div flex-xs="100" style="max-width:310px">
```

Comment:
The only time it should ever be applied is on that specific page, which is only used when running locally. I'd rather not muddy the stylesheet with a class for this singular use case.